packages/bruno-js/package.json

```diff
@@ -25,7 +25,6 @@
     "chai": "^4.3.7",
     "chai-string": "^1.5.0",
     "crypto-js": "^4.1.1",
```

Comment:
Maybe upgrade to 4.2.0 to remove the critical alert bring by the GHAS : https://github.com/advisories/GHSA-xwcq-pm8m-c4vf

Reply:
Sure let me bump it

Reply:
Done